gnocchiclient/gendoc.py

```diff
@@ -14,60 +14,26 @@
 
 from __future__ import absolute_import
 
-import sys
-
-from os_doc_tools import commands
+import fnmatch
 
+from cliff import sphinxext
```

Comment:
cliff and gnocchiclient should be separated by newline.

Reply:
Uhm, really? I'm fine to do that, but pep8 check is passing ok now.

Reply:
well the general styling is that it should be... similar to how os_doc_tools was not grouped with gnocchiclient previously.

```
native modules

3rd party moduels

in-repo modules
```

Reply:
If pep8 is passing, it's good enough, we should not nitpick too much on that :)

Reply:
i'm pretty sure our pep8 is only set to detect alphanumeric order seeing as i can change it to

>  from gnocchiclient import shell
> from os_doc_tools import commands
> import sys

and it passes... but meh.

Reply:
Well, in any case I added suggested change :).